docs/src/components/Markdown.css

```diff
@@ -1,22 +1,17 @@
-.Markdown p, pre, li {
-  margin-left: 30px;
-}
-
-.Markdown p, pre {
-  margin-bottom: 20px;
-  margin-top: 20px;
-}
-
-html[dir="rtl"] .Markdown p, pre, li {
-  margin-right: 30px;
-}
-
 .Markdown pre {
   background-color: rgb(236, 236, 236);
   overflow: auto;
   padding: 5px;
 }
 
+.Markdown ul + pre {
```

Comment:
@AlbertCarreras fyi this uses the `Adjacent sibling combinator` https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator

So we only indent the code (`pre`) when the previous child is a `ul`.

### Regular `pre`
![image](https://user-images.githubusercontent.com/127199/91059649-d6f37000-e5de-11ea-8790-43de968702ee.png)


### `pre` after a `ul`
![image](https://user-images.githubusercontent.com/127199/91059681-e2df3200-e5de-11ea-939f-f1a907e06ed7.png)